deployment/arm/deploy-region.sh

```diff
@@ -66,20 +73,32 @@ echo
 
 echo "Creating Regional Resource Group: $resourceGroupName"
 az group create \
---name $resourceGroupName \
---location $region
-
+    --name $resourceGroupName \
+    --location $region
+echo
 echo "Deploying regional resources to $resourceGroupName"
 az group deployment create \
---name "Urlist-$region-$(timestamp)" \
---resource-group $resourceGroupName \
---template-file region.json \
---parameters location=$region apimName=$apimName appServicePlanName=$appServicePlanName \
-frontendAppName=$frontendAppName backendAppName=$backendAppName appInsightsName=$appInsightsName storageActName=$storageActName
+    --name "Urlist-$region-$(timestamp)" \
+    --resource-group $resourceGroupName \
+    --template-file region.json \
+    --parameters location=$region apimName=$apimName appServicePlanName=$appServicePlanName \
+    backendAppName=$backendAppName appInsightsName=$appInsightsName storageActName=$storageActName
 
+echo
 echo "Configuring blob storage for static website hosting"
 az storage blob service-properties update \
---account-name $storageActName \
---static-website \
---index-document index.html \
---404-document index.html
\ No newline at end of file
+    --account-name $storageActName \
+    --static-website \
+    --index-document index.html \
+    --404-document index.html
+
+echo
+echo "Configuring app service authentication"
+az webapp auth update \
+    --name $backendAppName \
+    --resource-group $resourceGroupName \
+    --enabled true \
+    --action AllowAnonymous \
+    --twitter-consumer-key $TWITTER_KEY \
```

Comment:
Where is this $TWITTER_KEY & $TWITTER_SECRET set?

Reply:
Right now those would be environment variables that would need to be set prior to running the script.  The idea is they would be set in the AzDO pipeline variable group and make sure they are initialized before the shell script is called.